.github/workflows/action.yml

```
@@ -0,0 +1,50 @@
+name: Comment PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    name: Comment on Pull Request
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v2.1
+
+      - if: ${{ !contains(steps.changed-files.outputs.modified_files, 'noCopy.txt') }}
+        name: master comment
+        uses: thollander/actions-comment-pull-request@master
+        with:
+          message: |
+            Hey @${{ github.actor }}! Thank you for your pull request.
+            However, please do not open pull requests against the `master` branch, as stated in [CONTRIBUTING.md](../../CONTRIBUTING.md):
+            > push the changes you make to the dev branch (Jonas747/yagpdb/dev).
+            
+            If you still wish to keep this pull request open, please change the base branch.
+            For that, simply click the edit button next to the title of this pull request and select the new base;
+            To prevent conflicts, I suggest you first `git merge upstream/dev` on your local branch and push these new changes. Alternatively, `git rebase upstream/dev` on your local branch and then force-push.
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - if: ${{ contains(steps.changed-files.outputs.modified_files, 'noCopy.txt') }}
```

```suggestion
      - if: ${{ contains(steps.changed-files.outputs.modified_files, 'swearwords.go') }}
```
copy-paste mistake? :-)

---

hah, yes